docs/readthedocs/models/pv_model.md

```diff
@@ -276,41 +295,45 @@ b = (\cos(\phi) \cdot \cos(\delta)) \cdot (\sin(\omega_{2}) - \sin(\omega_{1}))
 $$
 
 $$
-E_{\mathrm{beam},S} = E_{\mathrm{beam},H} \cdot \frac{a}{b}
+G_{\mathrm{beam},S} = G_{\mathrm{beam},H} \cdot \frac{a}{b}
 $$
 
-**Please note:** $\frac{1}{180}\pi$ is omitted from these formulas, as we are already working with data in *radians*.
-
 *with*\
 **$\delta$** = the declination angle\
 **$\phi$** = observer's latitude\
 **$\gamma_{e}$** = slope angle of the surface\
 **$\omega_1$** = hour angle $\omega$\
 **$\omega_2$** = hour angle $\omega$ + 1 hour\
 **$\alpha_e$** = surface azimuth angle\
-**$E_{\mathrm{beam},H}$** = beam radiation (horizontal surface)
+**$G_{\mathrm{beam},H}$** = beam irradiance (horizontal surface)
+
+**Please note:**
+1. $\frac{1}{180}\pi$ is omitted from these formulas, as we are already working with data in *radians*.
+2. In contrast to the primary source {cite:p}`Duffie.2013`, radiance values instead of radiation values are used here, as described above.
 
 **Reference:**
 
 * {cite:cts}`Duffie.2013` p. 88
```

Comment:
This isn't plotted correct (does not need to be fixed in this PR).

Reply:
Nevermind, but maybe we should align our reference styles somewhen.

Reply:
Good idea...